pages/register.tsx

```diff
@@ -35,7 +38,16 @@ export default function Register() {
 
   const router = useRouter();
 
-  async function handleRegister(e: FormEvent) {
+  const { mutateAsync, isPending, error } = useMutation({
+    mutationKey: QueryKeys.REGISTER,
+    mutationFn: async (formData: typeof EMPTY_FORM_DATA) =>
+      await axios.post('/api/auth/register', formData),
+    onSuccess: () => userCreatedSuccesfully(),
```

Comment:
Tämän voi yksinkertaistaa näin
```suggestion
    onSuccess: userCreatedSuccesfully,
```